tests/sentry/api/endpoints/test_organization_missing_org_members.py

```diff
@@ -73,6 +73,19 @@ def setUp(self):
         not_shared_domain_author.save()
         self.create_commit(repo=self.repo, author=not_shared_domain_author)
 
+        self.invited_member = self.create_member(
```

Comment:
Should this be written as a test rather than implicitly tested in setup

Reply:
the first test fails if i add this without changing the concat to coalesce

Reply:
but i can explicitly add a check for it